.travis-docker.sh

```diff
@@ -8,13 +8,22 @@ default_branch=master
 default_hub_user=ocaml
 default_opam_version=2
 default_base_remote_branch=master
+beta_repository=git://github.com/ocaml/ocaml-beta-repository.git
 
 fork_user=${FORK_USER:-$default_user}
 fork_branch=${FORK_BRANCH:-$default_branch}
 hub_user=${HUB_USER:-$default_hub_user}
 opam_version=${OPAM_VERSION:-$default_opam_version}
 base_remote_branch=${BASE_REMOTE_BRANCH:-$default_base_remote_branch}
 
+if [ "$OCAML_BETA" = "enable" ]; then
+    if [ -z "${EXTRA_REMOTES}" ]; then
+        EXTRA_REMOTES="$beta_repository"
+    else
+        EXTRA_REMOTES="${EXTRA_REMOTES} $beta_repository"
+    fi
```

Comment:
This can portably be written:

```sh
EXTRA_REMOTES="$EXTRA_REMOTES${EXTRA_REMOTES:+ }$beta_repository"
```

Reply:
(see the handy [table in Posix)](https://pubs.opengroup.org/onlinepubs/009695399/utilities/xcu_chap02.html#tag_02_06_02)

Reply:
Wow I wasn't aware of this feature. Thanks!